packages/summary-table-react/documentation/SummaryTableExample.tsx

```diff
@@ -16,15 +16,49 @@ const SOFT_HYPHEN = "\u00AD";
 
 export const summaryTableExampleCode: CodeExample = `
 <SummaryTable
-    columnDescriptions={["Produkt", "Pris"]}
-    items={[
-        { label: "Person", value: "", bold: true },
-        { label: "Barne${SOFT_HYPHEN}forsikring", value: "200 kr/mnd" },
-        { label: "Behandlings${SOFT_HYPHEN}forsikring", value: "200 kr/mnd" },
-        { label: "Skade", value: "", bold: true },
-        { label: "Innboforsikring", value: "100 kr/mnd" },
-        { label: "Reiseforsikring", value: "100 kr/mnd" },
-    ]}
-    footer={{ label: "Total sum", value: "600 kr/mnd" }}
+    caption="Informasjon om produkter og pris i handlekurven"
+    header={["Produkt", "Pris"]}
+    body={
+        <tbody>
+            <tr>
+                <th scope="row">Uføre${SOFT_HYPHEN}pensjon</th>
+                <td>340,00 kr/mnd</td>
+            </tr>
```

Comment:
Dette føles veldig manuelt 🤔 Er heller ikke superfan av at vi setter stil rett på html-elementer (altså `tbody`, `tr`, etc.) i stedet for å bruke klasser.

Hva om man lager en `SummaryTableRow` som kan brukes her, som tar inn "overskrift" og innhold? Den kan ha en prop for å være grået ut, og ta inn klassenavn hvis man vil gjøre noe custom. Tenker også at det ikke bør være nødvendig å skrive inn `<tbody>` her. Innholdet vil vel alltid måtte være inne i en `tbody`, så da kan den defineres i `SummaryTable` i stedet 😊

Reply:
Godt forslag! Ja, jeg tror vi kan gjøre det som du sier